src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -117,13 +120,15 @@ private DownloadMapsWindow(
       progressPanel.download(pendingDownloads);
     }
 
+    pendingDownloads.addAll(ClientContext.DOWNLOAD_COORDINATOR.getDownloads().stream()
```

Comment:
Ahh, ok.  Any download in `pendingDownloads` will **not** be placed in the Available maps list.  So, I was concerned that a download would not get added to the Available list if it were cancelled.  It sounds like your filter does exactly what we want.  Thanks for clarifying.